cookbooks/prescribed_velocity/prescribed_velocity.cc

```diff
@@ -110,6 +110,51 @@ namespace aspect
   }
 
   /**
+   * This function retrieves the unit support points (in the unit cell) for the current element
+   * The DGP elements used when 'set Use locally conservative discretization = true' is enabled do
```

Comment:
"is enabled do" should probably be "is enabled to", but that still sounds funny. How about "...does not have support points"